packages/common/src/events/api/orval/api/orval-events.ts

```diff
@@ -277,6 +267,26 @@ export interface IOnlineMeeting {
   url: string;
 }
 
+export type ILocationGeometriesItem = { [key: string]: any };
```

Comment:
Minor/optional, I _believe_ we could alternatively use `Record<string, any>` here and for `ICreateLocationGeometriesItem`

Reply:
For generic types, this is how orval exports. I'm working on a next task to improve the validation on the geometries array, and this will be more explicit